### PR TITLE
autogenerateed protocol tests fixes 

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes
autogenerateed protocol tests fixes 

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/valory/skills/abstract_round_abci/skill.yaml [hunk 1]
```diff
@@ -145,7 +145,7 @@ dependencies:
   ipfshttpclient:
     version: ==0.8.0a2
   open-aea-cli-ipfs:
-    version: ==1.29.0
+    version: ==1.28.0.post1
```
Comment (DavidMinarsch) [inline packages/valory/skills/abstract_round_abci/skill.yaml:148]: Error

File: packages/valory/agents/test_ipfs/aea-config.yaml [hunk 2]
```diff
@@ -61,7 +61,7 @@ logging_config:
       propagate: true
 dependencies:
   open-aea-ledger-ethereum:
-    version: ==1.29.0
+    version: ==1.28.0.post1
```
Comment (DavidMinarsch) [inline packages/valory/agents/test_ipfs/aea-config.yaml:64]: Error

File: packages/valory/agents/test_abci/aea-config.yaml [hunk 3]
```diff
@@ -49,7 +49,7 @@ logging_config:
       propagate: true
 dependencies:
   open-aea-ledger-ethereum:
-    version: ==1.29.0
+    version: ==1.28.0.post1
```
Comment (DavidMinarsch) [inline packages/valory/agents/test_abci/aea-config.yaml:52]: Error

File: packages/valory/agents/registration_start_up/aea-config.yaml [hunk 4]
```diff
@@ -62,9 +62,9 @@ logging_config:
       propagate: true
 dependencies:
   open-aea-ledger-cosmos:
-    version: ==1.29.0
+    version: ==1.28.0.post1
   open-aea-ledger-ethereum:
-    version: ==1.29.0
+    version: ==1.28.0.post1
```
Comment (DavidMinarsch) [inline packages/valory/agents/registration_start_up/aea-config.yaml:67]: Error

File: packages/valory/agents/register_termination/aea-config.yaml [hunk 5]
```diff
@@ -69,7 +69,7 @@ logging_config:
       propagate: false
 dependencies:
   open-aea-ledger-ethereum:
-    version: ==1.29.0
+    version: ==1.28.0.post1
```
Comment (DavidMinarsch) [inline packages/valory/agents/register_termination/aea-config.yaml:72]: Error

File: packages/valory/agents/register_reset_recovery/aea-config.yaml [hunk 6]
```diff
@@ -59,9 +59,9 @@ logging_config:
       propagate: true
 dependencies:
   open-aea-ledger-cosmos:
-    version: ==1.29.0
+    version: ==1.28.0.post1
   open-aea-ledger-ethereum:
-    version: ==1.29.0
+    version: ==1.28.0.post1
```
Comment (DavidMinarsch) [inline packages/valory/agents/register_reset_recovery/aea-config.yaml:64]: Error

File: packages/valory/agents/register_reset/aea-config.yaml [hunk 7]
```diff
@@ -69,7 +69,7 @@ logging_config:
       propagate: true
 dependencies:
   open-aea-ledger-ethereum:
-    version: ==1.29.0
+    version: ==1.28.0.post1
```
Comment (DavidMinarsch) [inline packages/valory/agents/register_reset/aea-config.yaml:72]: Error

**Review by angrybayblade — DISMISSED** [review]
LGTM, the test failures are caused by connection timeouts not related to our codebase

File: docs/package_list.md [hunk 8]
```diff
@@ -1,51 +1,51 @@
...
-| connection/valory/p2p_libp2p_client/0.1.0                     | `bafybeidwcobzb7ut3efegoedad7jfckvt2n6prcmd4g7xnkm6hp6aafrva` |
+| protocol/open_aea/signing/1.0.0                               | `bafybeiclsbgrviyxbmi2vex5ze3dhr7ywohrqedebx26jozayxvroqtegq` |
+| protocol/valory/acn/1.1.0                                     | `bafybeieldogmtf3m4jdsvt4vvyay3jh54rjn3deasymfw43vz3o42vigmq` |
+| protocol/valory/http/1.0.0                                    | `bafybeifru3qs6udfzprax7jxktbsuzn7immfvi3scgfspifq3zdxwkgvnm` |
+| protocol/valory/ledger_api/1.0.0                              | `bafybeigqr4y3ykz3iulrcoqmji7hy3dxaoy7zmyyzff4ivpbubcpwdknai` |
+| protocol/valory/contract_api/1.0.0                            | `bafybeif32nchkgn6yet7e5gt4auhf7lsahxnj4t36kxbw55p3gi7qpeuxq` |
+| connection/valory/http_client/0.23.0                          | `bafybeigggahci7hq6tr3tyueatgkvgn73y4b3av2vk7vtr7jkeuwsqcteq` |
+| connection/valory/ledger/0.19.0                               | `bafybeicpxyoxez7lperltamvikxu6vzk2lhqakbivce4nzywyzoqbxoogm` |
+| connection/valory/p2p_libp2p_client/0.1.0                     | `bafybeiayoofxmj6z3pasn2akqj3udgq2ta2ar6mv6zoehstul2btvv3gqa` |
```
Comment (DavidMinarsch) [inline docs/package_list.md:51]: !!! https://github.com/valory-xyz/open-aea/blob/v1.29.0/packages/packages.json revert

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**